### ISSUE TITLE
For now an Alternative - Bitbucket

### ISSUE THREAD

**Issue by kuharan** [upend/IF_MS_BUYS_GITHUB_IMMA_OUT#63]
Have a look at this https://www.upguard.com/articles/github-vs-bitbucket 

Just in case anyone wants to have private repositories too.

**Comment by krixano** [comment]
I've always used Bitbucket for private repositories because they are free on there.

**Comment by licaon-kter** [comment]
#40

/Close this

**Comment by Serphentas** [comment]
Also #25 already talks about taking BitBucket into consideration. Close this.

**Comment by krixano** [comment]
Wow, you people are a little rude.

**Comment by Serphentas** [comment]
I personally don't have any power of influence on this repo, but as you can see for yourself there are already 28 open issues and this whole affair is not easily coped with. It is in the interest of all of us to keep this place as tidy as possible to avoid getting lost in a sea of suggestions (as good as they may be).

To make it clear, my comment does not aim to bash someone or an idea, but rather to help people gather all their suggestions in places where discussions are already taking place.

**Comment by krixano** [comment]
If this was in response to what I said, I was just meaning that instead of saying [blank] "already talks about" bitbucket. "Close this." You could have been more polite and said: I think [blank] already talks about Bitbucket, so perhaps this should be closed so all related discussion is located in one place? Politeness goes a long way, imo.

**Comment by TwinProduction** [comment]
My only problem with BitBucket is that the UI's disgusting compared to GitHub's.

**Comment by panchtatvam** [comment]
Gitlab anyone ?

**Comment by kuharan** [comment]
@panchtatvam yup checked out Gitlab yesterday. It has import repository (from GitHub) options too.

**Comment by vassudanagunta** [comment]
Hey guys. I love the work you've been doing. Not everyone loves mine. Please read Issue 114.

**Comment by vassudanagunta** [comment]
Hello,

The GitHub Evacuation Project has moved to GitLab (not an endorsement or even a final home). Your enthusiasm and contribution is still needed. Please check out [the new project home](https://gitlab.com/upend/github), and read [the project wiki](https://gitlab.com/upend/github/evac-guide/wikis) for info to get restarted.

Thank you!

vas
